addons/sale_coupon/models/coupon_program.py

```diff
@@ -23,9 +23,10 @@ def action_view_sales_orders(self):
             'name': _('Sales Orders'),
             'view_mode': 'tree,form',
             'res_model': 'sale.order',
+            'search_view_id': [self.env.ref('sale.sale_order_view_search_inherit_quotation').id, 'search'],
```

Comment:
why this, 'search' ? 
Seems like a leftover of filed_view_get ?

Reply:
Because it will allow the user to select `Confirmed` (well, `Sales Order` in this search view but it is the same) if they want to filter on confirmed sales and not quotation.
I didn't want to now list quotations too but without a way to filter them out.

Reply:
je parlais de 

'search_view_id': [self.env.ref('sale.sale_order_view_search_inherit_quotation').id, **'search'**],

Reply:
Ah, non je me suis basé sur [la doc](https://www.odoo.com/documentation/14.0/developer/reference/actions.html#window-actions-ir-actions-act-window) et les occurrences du code mais je viens de tester, on peut supprimer le `'search'` si tu veux, ça n'a à priori pas d'impacte (mais obligé de garder un array par contre)
Donc
```python
'search_view_id': [self.env.ref('sale.sale_order_view_search_inherit_quotation').id],
```